src/xcode/ENA/ENA/Source/HTTPClient/Service/__tests__/CachedRestServiceTests.swift

```diff
@@ -224,296 +224,4 @@ class CachedRestServiceTests: XCTestCase {
 		waitForExpectations(timeout: .short)
 	}
 	
-	// MARK: - Cache Policy tests
-	
-	func test_GIVEN_CachePolicyNoNetwork_WHEN_DefaultValueNoCacheIsSet_THEN_DefaultValueIsReturned() {
```

Comment:
Are these tests moved somewhere else?

Reply:
yes moved all of them to the new file `CachePolicyTests`